sdks/go/pkg/beam/io/spannerio/spanner_test.go

```diff
@@ -0,0 +1,23 @@
+package spannerio
+
+import (
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"testing"
+)
+
+type TestDto struct {
+	One string `spanner:"one"`
+	Two int    `spanner:"two"`
+}
+
+func TestColumnsFromStructReturnsColumns(t *testing.T) {
+	// arrange
+	// act
+	cols := columnsFromStruct(reflect.TypeOf(TestDto{}))
+
+	// assert
+	assert.NotEmpty(t, cols)
+	assert.Equal(t, "one", cols[0])
+	assert.Equal(t, "two", cols[1])
```

Comment:
Prefer using [cmp](https://pkg.go.dev/github.com/google/go-cmp/cmp) instead and not introducing an assert framework to the SDK.

Reply:
Done